### PR TITLE
Unified x86-x64 bootstrapper

### PR DESCRIPTION
**Pull request by sxmanton**
Includes changes requested in [POS-555] and [POS-556]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,12 +21,9 @@ A simple installer which enables [Transport Layer Security](https://en.wikipedia
...
 
 ## Download and Installation
-Make sure to use the version which matches your architecture: 
+The latest version can be downloaded [here](../../releases/latest).
 
-* [x86](../../releases/latest) for 32-bit versions of Windows
-* [x64](../../releases/latest) for 64-bit versions of Windows. 
-
-To install, simply run the installer and accept the license agreement. You must have administrative priveledges. You will be prompted to restart your computer in order for the changes to take effect.
+To install, simply run the installer and accept the license agreement. You must have administrative priveledges. You may be prompted to restart your computer in order for the changes to take effect.
```
Comment (mmercurio) [inline README.md]: Why the change from "you will be prompted to restart" to "you *may* be prompted to restart"?

File: README.md [hunk 2]
```diff
@@ -37,10 +35,14 @@ The installer:
 * Writes a registry value which makes .NET 4.x code select the strongest enabled security protocol by default.
 * Writes registry values to explicitly enable TLS 1.2 for [WinSSL](https://msdn.microsoft.com/en-us/library/windows/desktop/aa380123(v=vs.85).aspx).
 
+The patcher will appear in Add/Remove Programs. Uninstalling the patcher will delete the registry values it created, potentially disabling TLS 1.2. Uninstalling the patcher will not uninstall .NET 4.6. 
+
```
Comment (mmercurio) [inline README.md]: If the patcher modifies an existing registry value, what happens on uninstall?  This makes it sound like the value will be deleted.

File: LevelUp.Integrations.TlsPatcher.Bootstrapper/LevelUp.Integrations.TlsPatcher.Bootstrapper.wixproj [hunk 3]
```diff
@@ -83,15 +78,15 @@
...
+  <!-- Arbitrary event that ensures GitVersionTask loads GitVersion environment variables -->
   <Target Name="GetAssemblyVersion" BeforeTargets="EnsureWixToolsetInstalled">
     <Message Text="Triggering GetAssemblyVersion..." Importance="High" />
   </Target>
   <Target Name="SharedProperties">
     <PropertyGroup>
       <BuildScriptSourceDir>$(SolutionDir)BuildScripts\</BuildScriptSourceDir>
       <BuildScriptSourceDirSignBootstrapper>$(BuildScriptSourceDir)SignBootstrapper.ps1</BuildScriptSourceDirSignBootstrapper>
-      <BuildTargetFilename>$(TargetName)-$(GitVersion_NuGetVersion)_$(Platform)$(TargetExt)</BuildTargetFilename>
+      <BuildTargetFilename>$(TargetName)-$(GitVersion_NuGetVersion)_x86_x64$(TargetExt)</BuildTargetFilename>
```
[the remark this answers is not in the document]
Comment (sxmanton) [reply LevelUp.Integrations.TlsPatcher.Bootstrapper/LevelUp.Integrations.TlsPatcher.Bootstrapper.wixproj]: ⛔️

**Comment by mmercurio** [comment]
LGTM!  Thank you!
